AnoopExercises/Sources/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  PhotoCollectionViewCell.swift
+//  AnoopExercises
+//
+//  Created by Anoop Subramani on 17/02/22.
+//
+
+import UIKit
+
+class PhotoCollectionViewCell: UICollectionViewCell {
+    // MARK: - Views
+
+    private weak var imageView: UIImageView!
+    private weak var spinner: UIActivityIndicatorView!
+
+    // MARK: - Variables
+
+    static let cellIdentifier = "PhotoCell"
+    var photoDescription: String?
+    override var accessibilityLabel: String? {
+        get {
+            return photoDescription
+        }
+        set {}
+    }
+
+    override var accessibilityTraits: UIAccessibilityTraits {
+        get {
+            return super.accessibilityTraits.union([.image, .button])
+        }
+        set {}
+    }
+
+    override var isAccessibilityElement: Bool {
+        get {
+            return true
+        }
+        set {}
+    }
```

Comment:
These can be set up in `update` function?

Reply:
yes i just had to remove override and relocate them into update function, this gives the same result, ill make the changes